ui/vty-app/Glue.hs

```diff
@@ -187,7 +200,11 @@ knitFaceToUI UiFace{..} KnitFace{..} putPass =
             Knit.ValueHash h -> Right $ pretty h
             _ -> Left "Unrecognized return value"
       UiFee{} ->
-        Just . UiFeeCommandResult . UiFeeCommandSuccess $ "not implemented"
+        Just . UiFeeCommandResult . either UiFeeCommandFailure UiFeeCommandSuccess $
+          fromResult result >>= fromValue >>= \case
+            Knit.ValueCoin c -> Right $ pretty c
+            _ -> Left "Unrecognized return value"
```

Comment:
It's better to `show` the value:
`v -> Left $ "Unrecognized return value: " <> show v`